.github/workflows/conformance-aks-v1.11.yaml

```diff
@@ -114,11 +112,103 @@ jobs:
             src:
               - '!(test|Documentation)/**'
 
+  setup-report:
+    runs-on: ubuntu-latest
```

Comment:
Do we always use latest? Shall we align to LTS (22.04)?

Reply:
Yes, all CI uses this